app/assets/stylesheets/common/base/sidebar.scss

```diff
@@ -167,4 +213,13 @@
       margin-right: 0;
     }
   }
+  .select-kit.multi-select .multi-select-header .formatted-selection {
+    display: none;
+  }
+  .modal-inner-container .select-kit {
+    width: 60px;
+  }
+  .select-kit.is-expanded .select-kit-body {
+    width: 220px !important;
```

Comment:
What is the reason we need to use `!important` here? Also will hardcoding the width be a problem on mobile devices?

Reply:
!important is added because selectkit it adding inline CSS:

<img width="1898" alt="Screenshot 2023-05-25 at 1 29 48 pm" src="https://github.com/discourse/discourse/assets/72780/cae08bce-cd01-4a06-ab42-60695d7cfe68">

Without !important it takes width from parent element:
<img width="412" alt="Screenshot 2023-05-25 at 1 32 19 pm" src="https://github.com/discourse/discourse/assets/72780/e7e75d74-e82a-483d-9ee3-0739ea76a664">

I feel that 220px is reasonable for most mobile devices, and it allows the longest icon to be properly displayed:
<img width="333" alt="Screenshot 2023-05-25 at 1 35 32 pm" src="https://github.com/discourse/discourse/assets/72780/60693f67-598d-4716-b24a-b7858ae89df1">

Reply:
It works on desktop fine, but min width is not applied on mobile devices. We have hard-coded 220px there as well.

https://github.com/discourse/discourse/blob/a22f83b7f78541bf9dccbdf8e71568dbbebc80bd/app/assets/javascripts/select-kit/addon/components/select-kit.js#L961

Reply:
Can you rely on the selectKit option on desktop and only use the width override on mobile?